Cake.DoInDirectory/Cake.DoInDirectory.csproj

```diff
@@ -20,7 +20,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Cake.Core" Version="0.33.0" PrivateAssets="All" />
+    <PackageReference Include="Cake.Core" Version="1.1.0" PrivateAssets="All" />
```

Comment:
@pitermarx There are no breaking changes to the API between Cake `1.0.0` and Cake `1.1.0`, and your addin doesn't rely on anything added to `1.1.0` so it would be better if your addin targets `1.0.0` for broader reach - i.e. Your addin will be usable for devs using either Cake `1.0.0` or `1.1.0`.

Reply:
@pitermarx gave me greenlight to maintain this project going forward.

I update the PR to target Cake 1.1